test/image/index.html

```diff
@@ -5,6 +5,7 @@
 <script type="text/javascript" src="../plotly.js/dist/extras/mathjax/MathJax.js?config=TeX-AMS-MML_SVG"></script>
 <script type="text/javascript" src="../plotly.js/build/plotly.js" charset="utf-8"></script>
 <script type="text/javascript" src="../plotly.js/dist/plotly-geo-assets.js" charset="utf-8"></script>
+<script type="text/javascript" src="../plotly.js/test/image/strict-d3.js" charset="utf-8"></script>
```

Comment:
Great idea here!

First, I was thinking adding this to the image test index might lead to some very annoying debugging sessions when things go wrong. But, by adding it to the test-dashboard as well essentially make my concerns disappear  (except for devs that don't use the test-dashboard cc @rreusser @monfera).

Alternatively, we could mock the `d3.selection.prototype` only in the [`ie9_test.js`](https://github.com/plotly/plotly.js/blob/master/test/jasmine/bundle_tests/ie9_test.js) bundle test and test each ie9 falback one by one.

It comes down to choosing between a solution with great coverage (yours) and a less intrusive solution that might lead to less :face_with_head_bandage: down the road. I'm ok with both.